src/test/ui/allocator/not-an-allocator.stderr

```diff
@@ -5,7 +5,7 @@ LL | static A: usize = 0;
    | ^^^^^^^^^^^^^^^^^^^^ the trait `GlobalAlloc` is not implemented for `usize`
    |
    = note: required by `std::alloc::GlobalAlloc::alloc`
-   = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
+   = note: this error originates in the attribute macro `global_allocator` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
Unrelated to this PR, but would it be possible to only show "run with -Z macro-backtrace" in nightly builds, the same way the compiler only suggests `#![feature]` on nightly?

Reply:
For months I've thought of suggesting this but kept forgetting, so thanks for suggesting it!

Reply:
This can be a very useful option for debugging macros, so I think it's good to make people aware of it. Unlike `#![feature]`s, this option is something can be enabled just for local testing, without requiring that anyone else use a nightly compiler.

Reply:
@Aaron1011's observation is exactly the reason I didn't gate that suggestion on being in nightly.